Classes/Private/Manager/OCTManager.m

```diff
@@ -27,6 +28,7 @@ @interface OCTManager () <OCTToxDelegate, OCTSubmanagerDataSource>
 @property (strong, nonatomic, readwrite) OCTSubmanagerChats *chats;
 @property (strong, nonatomic, readwrite) OCTSubmanagerAvatars *avatars;
 @property (strong, nonatomic, readwrite) OCTSubmanagerFiles *files;
+@property (strong, nonatomic, readwrite) OCTSubmanagerCalls *managerCalls;
```

Comment:
Here you have `managerCalls`, but in all other places `callsManager`. Also I'm wondering why you want to name it `callsManager`? Why not `calls` to be consistent will all other submanagers?